Business resume/contact.html

```diff
@@ -0,0 +1,113 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="homepage.css">
+    <link rel="stylesheet" href="contact.css">
+    <link rel="stylesheet" href="resume.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Business Resume</title>
+</head>
+<body>
+    <header>
+        <div class="header-left">
+            <div class="header-box">
+                <div class="square">
+                </div>
+                <a href="index.html">Maya Nelson</a>
+            </div>
+            
+            <span>/</span>
+            
+            <a class="title-header" href="index.html">PROJECT MANAGER</a>
+        </div>
+
+        <div class="header-right">
+            <input id="toggle" type="checkbox"></input>
+
+            <label for="toggle" class="hamburger">
+            <div class="top-bun"></div>
+            <div class="meat"></div>
+            <div class="bottom-bun"></div>
+            </label>
+
+            <div class="nav">
+            <div class="nav-wrapper">
+            <nav>
+                <ul>
+                    <li><a href="index.html">About me</a></li>
+                    <li><a href="resume.html">Resume</a></li>
+                    <li><a href="#">Projects</a></li>
+                    <li><a href="contact.html">Contact</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <main>
+        <div class="title-main">
+            <div class="square">
+                    </div>
+            <h1>Let's talk</h1>
+        </div>
+       <form>
```

Comment:
Do you remember a couple of weeks ago we talked about form method, `GET` and `POST`? Which one do you think would be suitable for this contact form?